### PR TITLE
Event tweaks

### PR DESCRIPTION
**Pull request by YashoSharma**
- [x] clean up work on the eventsapi

### CODE REVIEW

File: api/database.js [hunk 1]
```diff
@@ -5,6 +5,7 @@ var milliseconds = require('ms');
 
 var KNEX_CONFIG = {
 	client: 'mysql',
+	debug: true,
```
Comment (tommypacker) [inline api/database.js]: No

File: api/v1/services/EventService.js [hunk 2]
```diff
@@ -58,7 +61,4 @@ module.exports.createEvent = function (params) {
                     });
             });
         })
```
Comment (tommypacker) [inline api/v1/services/EventService.js]: ;

**Comment by YashoSharma** [comment]
Closing this, its addressed in #113

**Comment by YashoSharma** [comment]
Closing this, its addressed in #113

**Pull request closed without merging**